aspnetcore/fundamentals/websockets.md

```diff
@@ -148,13 +139,42 @@ When accepting the WebSocket connection before beginning the loop, the middlewar
 
 ::: moniker range=">= aspnetcore-2.2"
 
-### Handle client disconnects
+## WebSocket lifetime
+
+The WebSocket that is received from `AcceptWebSocketAsync` lives only as long as the middleware pipeline for the request is still running. When the middleware pipeline ends, the socket is closed. If your code continues to try to send and receive messages, a `WebSocketException` exception is raised with the following message:
```

Comment:
The WebSocket that is received from `AcceptWebSocketAsync` lives only as long as the middleware pipeline for the request is still running. When the middleware pipeline ends, the socket is closed.  The following `WebSocketException` exception is thrown when attempting to send and receive messages after the socket is closed: